src/ds5/ds5-options.cpp

```diff
@@ -667,6 +667,12 @@ namespace librealsense
 
     void auto_exposure_limit_option::set(float value)
     {
+        if (value < get_range().min || value > get_range().max)
```

Comment:
Use built-in method for checking and reporting the range:
```cpp
if (!is_valid(value))
            throw invalid_value_exception(to_string() ....
```

Reply:
fixed